cfgov/unprocessed/js/apps/financial-well-being/fwb-results.js

```diff
@@ -0,0 +1,72 @@
+/* eslint no-extra-semi: "off" */
+'use strict';
+
+var Analytics = require( '../../modules/Analytics' );
+var Expandable = require( '../../organisms/Expandable' );
+
+var expandableDom = document.querySelectorAll( '.content .o-expandable' );
+var expandable;
+if ( expandableDom ) {
+  for ( var i = 0, len = expandableDom.length; i < len; i++ ) {
+    expandable = new Expandable( expandableDom[i] );
+    expandable.init();
+  }
+}
+
+function init() {
+
+  var toggleButtons = document.querySelectorAll(
+    '.comparison-chart_toggle-button'
+  );
+
+  function switchComparisons( category ) {
+    var allCategories = document.querySelectorAll( '.comparison_data-point' );
+    var selectedButtonClass = 'comparison-chart_toggle-button__selected';
+
+    // Hide all categories ...
+    [].forEach.call( allCategories, function( el ) {
+      el.style.display = 'none';
```

Comment:
Better to add/remove classes so debugging in the output markup is easier and styling is done consistently. Though this would mean needing a modifier class for hidden, which you may not want.

Reply:
Actually you can add/remove `u-hidden`, that'd be the way to do it.

Reply:
Thanks for this, updated in the next PR.